### DISCUSSION TITLE
Боевой АИ сталкеров

### DISCUSSION THREAD

**Discussion by AshHex** [OGSR/OGSR-Engine#456] [General]
Нет желания подправить боевой АИ сталкеров? Например как в Grand Rendering Mod там нпс-враги шустро передвигаются, быстро атакуют, уходят за укрытия и тд. Или это поломает совместимость при адаптации других модов?

**Comment by I-AmDead** [comment]
Ну так это не движок, хочешь продвинутый аи - бери и адаптируй аи пак, хочешь чтобы нпс быстро передвигались - ускоряй их

**Reply by AshHex** [reply]
АИ пак я ставил, но это не то, я имею ввиду именно передвижение нпс во время боя, спрашивал у автора Grand Rendering Mod использовал ли он какие-либо скрипты, он ответил что это полностью движковая правка, судя по всему взятая из билдов

**Reply by I-AmDead** [reply]
Какой толк от этого? Весь мод готовы в движок запихать уже, блин

**Reply by AshHex** [reply]
мое дело предложить, ваше дело отказаться, в любом случае спасибо за движок и удачи в разработке

**Reply by I-AmDead** [reply]
пожалуйста! но затея с внедрением в движок всего подряд что можно сделать скриптами - такая себе, мягко говоря

**Reply by AshHex** [reply]
как я понял, перенос в движок таких фич, делает не возможность их растаскивания по другим модам без исходников движка (как в том же просекторе)

**Reply by I-AmDead** [reply]
Считаю такое свинством, если бы каждый так мыслил, то сидели бы в луже оригинала тч до сих пор
